specification.md

```diff
@@ -47,7 +47,7 @@ _If you are accessing the above meta-schema links **from a web browser**, you wi
 Migrating from older drafts
 -------------
 
-The release notes discuss the changes impacting users and implementors:
```

Comment:
Both spellings are correct, and I'm sure we use "implementors" elsewhere so I'd prefer not to change it.